src/main/java/uk/gov/justice/digital/hmpps/keyworker/dto/RoleAssignmentStats.java

```diff
@@ -0,0 +1,55 @@
+package uk.gov.justice.digital.hmpps.keyworker.dto;
+
+import io.swagger.annotations.ApiModelProperty;
+import lombok.*;
+import uk.gov.justice.digital.hmpps.keyworker.rolemigration.RoleAssignmentsService;
+
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@ToString
```

Comment:
do you need these?

Reply:
we use the getters but have removed other annotations